<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GkProj</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="RasterWidget" name="rasterWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionForegroundColor"/>
    <addaction name="actionBackgroundColor"/>
    <addaction name="actionGrid"/>
    <addaction name="separator"/>
    <addaction name="actionPoint"/>
    <addaction name="actionLine"/>
    <addaction name="actionCircle"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionOutline"/>
    <addaction name="actionProperties"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="outlineDock">
   <property name="windowTitle">
    <string>O&amp;utline</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="outlineDockContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="propertiesDock">
   <property name="windowTitle">
    <string>P&amp;roperties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="propetriesDockContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionForegroundColor"/>
   <addaction name="actionBackgroundColor"/>
   <addaction name="actionGrid"/>
   <addaction name="separator"/>
   <addaction name="actionPoint"/>
   <addaction name="actionLine"/>
   <addaction name="actionCircle"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="draw-calligraphic">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Point</string>
   </property>
   <property name="toolTip">
    <string>Draw a point</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="draw-line">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Line</string>
   </property>
   <property name="toolTip">
    <string>Draw a line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionOutline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="view-list-text">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Outline</string>
   </property>
   <property name="toolTip">
    <string>Toggle outline dock</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="document-properties">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Properties</string>
   </property>
   <property name="toolTip">
    <string>Toggle properties dock</string>
   </property>
  </action>
  <action name="actionForegroundColor">
   <property name="icon">
    <iconset theme="color-picker">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Foreground color</string>
   </property>
   <property name="toolTip">
    <string>Choose foreground color</string>
   </property>
  </action>
  <action name="actionBackgroundColor">
   <property name="icon">
    <iconset theme="kdenlive-add-color-clip">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Background color</string>
   </property>
   <property name="toolTip">
    <string>Choose background color</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="grid-rectangular">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Grid</string>
   </property>
   <property name="toolTip">
    <string>Toggle grid</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="draw-circle">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Circle</string>
   </property>
   <property name="toolTip">
    <string>Draw a circle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RasterWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>rasterwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outlineDock</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionOutline</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>propertiesDock</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionProperties</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOutline</sender>
   <signal>toggled(bool)</signal>
   <receiver>outlineDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>663</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionProperties</sender>
   <signal>toggled(bool)</signal>
   <receiver>propertiesDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>663</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPoint</sender>
   <signal>changed()</signal>
   <receiver>MainWindow</receiver>
   <slot>chooseShape()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCircle</sender>
   <signal>changed()</signal>
   <receiver>MainWindow</receiver>
   <slot>chooseShape()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLine</sender>
   <signal>changed()</signal>
   <receiver>MainWindow</receiver>
   <slot>chooseShape()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseShape()</slot>
 </slots>
</ui>
